os/cluster-architectures.md

```diff
@@ -86,38 +96,42 @@ Following the guide for each of the [supported platforms](https://coreos.com/doc
 |------|-----------|-------------|------------|
 | Low | Development/Testing | Minutes | No |
 
-When you're first getting started with Container Linux, it's common to frequently tweak your cloud-config which requires booting/rebooting/destroying many machines. Instead of being slowed down and distracted by generating new discovery URLs and bootstrapping etcd, it's easier to start a single etcd node.
+When getting started with Container Linux, you may find yourself booting/rebooting/destroying many machines. Instead of slowing down and distracting yourself, generating new discovery URLs and bootstrapping etcd. Instead, start a single etcd node and build your cluster around that.
```

Comment:
When getting started with Container Linux, it's common to frequently boot, reboot, and destroy machines while tweaking your configuration. To avoid the need to generate new discovery URLs and bootstrap etcd, start a single etcd node, and build your cluster around it.